### PR TITLE
Small doc fixes

### PR DESCRIPTION
**Pull request by nvie**

### CODE REVIEW

File: docs/pages/api-reference/liveblocks-client.mdx [hunk 1]
```diff
@@ -245,7 +245,7 @@ room.history.undo();
 
 ### Room.getOthers
 
-Gets all the other users in the Room.
+Gets all the other `User`s in the Room.
```
Comment (nvie) [inline docs/pages/api-reference/liveblocks-client.mdx:248]: I would love to add a dedicated page in the docs to document the `User` type (the return value of `room.getSelf()` and `room.getOthers()`. It has quite a few properties (like `connectionId`, `presence`, `id`, `isReadOnly`, etc.)

Those will be good to be able to explain and reference more clearly.

**Comment by andreaselia** [comment]
Since this is a general doc-fix PR, and rather than creating my own, perhaps you could apply a missing curly brace fix to the code on this page?

https://liveblocks.io/docs/api-reference/liveblocks-client#Room.subscribe.others

The below `if` statement is missing the closing curly brace.

```javascript
if (event.type === "reset") {
```

**Comment by nvie** [comment]
Thanks, @andreaselia! 🙏

**Comment by andreaselia** [comment]
No problemo @nvie, thanks for updating that 👍 

There's another one on this page if you'd like to update it as part of this PR too.

https://liveblocks.io/docs/api-reference/liveblocks-zustand

"with ZUstand bindings" -> "with Zustand bindings"

**Comment by nvie** [comment]
Thanks @andreaselia — you're on a roll! 🔥

**Comment by nvie** [comment]
Pinging @CTNicholas for a quick review.

**Review by CTNicholas — APPROVED** [review]

**Comment by CTNicholas** [comment]
Thanks for the changes @andreaselia, @nvie!

**Pull request merged**